Atomic/backendutils.py

```diff
@@ -41,9 +41,12 @@ def get_backend_and_image_obj(self, img, str_preferred_backend=None):
         # Check preferred backend first
         if str_preferred_backend:
             be = self.get_backend_from_string(str_preferred_backend)
-            img_obj = be.has_image(img)
-            if img_obj:
-                return be, img_obj
+            try:
```

Comment:
I originally intended this patch to replicate behavior in the old (separated) backend, where if the docker daemon wasn't running, the commands would just display blank for docker images/containers and not error.

With the exception of generate (which requires gomtree) every other image/container command supports ostree images/containers. I could pass the option to ignore NoDaemon errors, but for all these commands, there is no "error" for docker daemon not running. Alternatively I could display a warning?